src/modules/claim/entities/roleClaim.entity.ts

```diff
@@ -83,6 +91,10 @@ export class RoleClaim implements IRoleClaim {
   @Field()
   @Column()
   namespace: string;
+
+  @Field({ nullable: true })
+  @Column({ nullable: true })
+  requestCompletionRedirectUrl?: string;
```

Comment:
```suggestion
  /**
   * This is location to which the client can be redirected after the request/exchange
   * TODO: Confirm the extent to which this is similar to OpenID Connect redirect_uri
   */
  @Field({ nullable: true })
  @Column({ nullable: true })
  redirectUri?: string;
```

Reply:
I am thinking right now that is not exactly the `redirectionUri` but `requestOriginUri`. One of the potential use cases is redirection, but not necessary the only one. What you think?

Reply:
Although both points are valid I am inclined to `redirectUri`, because the purpose of this property is to redirect. It is just happened that this value is set to origin

Reply:
I agree with Dmitry, I think that the property is currently coming from HTTP Origin header, but we could set it in another way in the future if necessary